spring-boot-project/spring-boot-docs/src/main/asciidoc/howto.adoc

```diff
@@ -1167,7 +1167,7 @@ work. To use the Jackson XML renderer, add the following dependency to your proj
 	</dependency>
 ----
 
-If Jackson's XML extension is not available, JAXB (provided by default in the JDK) is
+If Jackson's XML extension is not available, JAXB (provided by default in the JDK 8) is
```

Comment:
Rather than focusing on JDK 8 and now that this is an optional feature, I'd prefer if we rephrase that section along the lines of "if available".